steps/parameterList.ts

```diff
@@ -5,3 +5,11 @@ export const parameterDescriptions = {
   mulOnly: 'Skip ZIM files for individual languages',
   createMul: 'Create a ZIM file with all languages',
 }
+
+export const applyParameterConstraints = (argv): boolean => {
+  const regex = /^(?:[a-zA-Z0-9]+,)*[a-zA-Z0-9]+$/ // ensure that string follow this format 'en,ar,fr'
```

Comment:
Doesn't the scraper support variants as language, e.g. `pt_BR` (I'm not 100% sure, so please check and do not hesitate to speak up if it looks like I'm wrong)

Reply:
Your regex does not seems to match these variants, or am I wrong?
Plus the regex does not ensures at all that we have only two chars or two chars + underscore + two chars.
And now you need to adapt it to allow blank characters everywhere.